stable/mariadb/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mariadb
-version: 6.0.1
+version: 6.1.1
```

Comment:
```suggestion
version: 6.1.0
```

Reply:
Sorry 'bout that one - should be fixed now...